lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -1535,6 +1535,12 @@ a.fade-cover {
 
 #unenroll-modal {
   margin-top: -60px;
+
+  .modal-form-error {
+    background: tint($red, 95%);
+    margin-left: $baseline;
```

Comment:
use margin: 0 $baseline;

Reply:
I want the inherited `margin-top`/`margin-bottom` values to apply, so I think I will need to leave this as is.